src/main/java/cuchaz/enigma/translation/mapping/serde/EnigmaMappingsReader.java

```diff
@@ -122,25 +127,40 @@ private int countIndentation(String line) {
 		return indent;
 	}
 
-	private MappingPair<?, EntryMapping> parseLine(@Nullable Entry<?> parent, String line) {
+	private String tryReadComment(String line) {
+		String[] tokens = line.substring(countIndentation(line)).trim().split("\\s", 2);
```

Comment:
`trim()` should do the work that `substring` is doing here -- potentially misunderstanding, but maybe that could be removed?